### ISSUE TITLE
using name, label and shortnames 

### ISSUE THREAD

**Issue by dreirik** [foss-events/website#63]
inside the list on the front-page of // foss.events there should be used "label" for the name. 

on the subpages, the title should be {name} ({shortname})

however, this needs an short update of the db first. I will post here once the db is ready for this change.

**Comment by weeman1337** [comment]
* shortname 1.5rem in `small`
* up to three chars in shortname → goes to the same line, else line break
* 2rem margin bottom
* left aligned

**Comment by Moini** [comment]
Yes, I found this not be described consistently (in the file that lists the variables, it says that 'name' is supposed to be displayed on the event detail page, which is how I formatted my entry, I didn't check the template if that was correct... Now it says 'Kielux' in the event page, while I thought that would be used only for the preview list, and the name would be visible there). 

I think that 'name' should be used both on the detail page and in the main list. Why would there need to be different names?

Label isn't needed, or should be redefined as slug (i.e. internal shortcut name to be used for urls and ids etc. that are not user-facing).

Shortname, in my understanding of the term, is what is currently used as 'label'.

I was just about to start working on the Name / Label differentiation when I found this report, and saw that you two might not think along the same lines, so I wanted to ask first :)
